### PR TITLE
Fix Filterviews

### PR DESCRIPTION
**Pull request by berfarah**
I was unable to test this before, not having a sandbox for it. This is
also a great reason to have tests :)

It's now working \o/

### CODE REVIEW

File: src/commands/filterviews.js [hunk 1]
```diff
@@ -16,7 +18,7 @@ function generateFilterViews({ serviceKey, spreadsheetId, sheetname, range, dir,
           return filterViews.clear(authClient, spreadsheetId, sheet).then(function () {
             console.log(`[${ sheetname }] cleared existing filter views`);
             return Promise.resolve(sheet);
-          }, function() {
+          }, function(err) {
```
Comment (kylealwyn) [inline src/commands/filterviews.js]: now may be a good time to move these to `.catch` blocks

Comment (berfarah) [reply src/commands/filterviews.js]: Yeah, really should. I guess this is a good time to do it.

**Comment by berfarah** [comment]
Since I was fixing stuff I figured I'd just fix a bunch of shit.

**Comment by berfarah** [comment]
I'd suggest making this `2.2.0` because of the addition of nested objects.

Commit #1 is a patch, #2 is a patch, #3 is a minor and #4 is probably not even a patch

File: src/commands/bundles.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+const console = require('../console');
```
Comment (kylealwyn) [inline src/commands/bundles.js]: not sure about overriding global `console`. now i can't run any simple console.logs with my own values for output. how about `logger`?

Comment (berfarah) [reply src/commands/bundles.js]: Yeah, I had that gut feeling, but I think I was feeling a little lazy. I can rename it to logger.

Comment (berfarah) [reply src/commands/bundles.js]: Updated

**Review by kylealwyn — APPROVED** [review]

**Pull request merged**